frontend/packages/core/src/Input/select.tsx

```diff
@@ -72,18 +70,21 @@ const Select: React.FC<SelectProps> = ({
   };
 
   React.useEffect(() => {
-    onChange(options[selectedIdx].value || options[selectedIdx].label);
+    onChange(options[selectedIdx]?.value || options[selectedIdx].label);
```

Comment:
should it be `?.label` all throughout?

Reply:
every SelectOption should enforce that label is required. Removed the optional chaining of this below